plinth/modules/bind/__init__.py

```diff
@@ -0,0 +1,119 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Plinth module to configure BIND server
+"""
+
+from django.urls import reverse_lazy
+from django.utils.translation import ugettext_lazy as _
+
+from plinth import actions
+from plinth import action_utils
+from plinth import cfg
+from plinth import frontpage
+from plinth import service as service_module
+from plinth.views import ServiceView
+
+
+version = 1
+
+depends = ['apps']
+
+title = _('Domain Name Server \n (BIND)')
+
+service = None
+
+managed_services = ['bind9']
+
+managed_packages = ['bind9', 'bind9utils', 'bind9-doc']
+
+description = [
+    _('BIND is open source software that enables you to publish your Domain '
+      'Name System (DNS) information on the Internet, and to resolve '
+      'DNS queries for your users.'),
+
+    _('BIND is by far the most widely used DNS software on the Internet, '
+      'providing a robust and stable platform on top of which organizations'
+      ' can build distributed computing systems with the knowledge that those '
+      'systems are fully compliant with published DNS standards.')
+]
+
+
+def init():
+    """Intialize the BIND module."""
+    menu = cfg.main_menu.get('apps:index')
```

Comment:
I know this is still a work-in-progress, but I thought to make some suggestions.

Since DNS will work for the most part automatically / in the background, would it be better to put it under the System menu? In which case, we probably don't need a front page shortcut for it either.